x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/signals_alert_type.ts

```diff
@@ -88,56 +92,34 @@ export const signalsAlertType = ({ logger }: { logger: Logger }): SignalAlertTyp
 
       try {
         logger.debug(`Starting signal rule "id: ${alertId}", "ruleId: ${ruleId}"`);
-        if (process.env.USE_REINDEX_API === 'true') {
-          const reIndex = buildEventsReIndex({
-            index,
-            from,
-            to,
-            // TODO: Change this out once we have solved
-            // https://github.com/elastic/kibana/issues/47002
-            signalsIndex: process.env.SIGNALS_INDEX || DEFAULT_SIGNALS_INDEX,
-            severity,
-            description,
-            name,
-            timeDetected: new Date().toISOString(),
-            filter: esFilter,
-            maxDocs: maxSignals,
-            ruleRevision: 1,
-            id: alertId,
-            ruleId,
-            type,
-            references,
-          });
-          const result = await services.callCluster('reindex', reIndex);
-          if (result.total > 0) {
-            logger.info(
-              `Total signals found from signal rule "id: ${alertId}", "ruleId: ${ruleId}" (reindex algorithm): ${result.total}`
-            );
-          }
-        } else {
-          logger.debug(
-            `[+] Initial search call of signal rule "id: ${alertId}", "ruleId: ${ruleId}"`
+        logger.debug(
+          `[+] Initial search call of signal rule "id: ${alertId}", "ruleId: ${ruleId}"`
+        );
+        const noReIndexResult = await services.callCluster('search', noReIndex);
+        if (noReIndexResult.hits.total.value !== 0) {
+          logger.info(
+            `Total signals found from signal rule "id: ${alertId}", "ruleId: ${ruleId}": ${noReIndexResult.hits.total.value}`
           );
-          const noReIndexResult = await services.callCluster('search', noReIndex);
-          if (noReIndexResult.hits.total.value !== 0) {
-            logger.info(
-              `Total signals found from signal rule "id: ${alertId}", "ruleId: ${ruleId}": ${noReIndexResult.hits.total.value}`
-            );
-          }
+        }
 
-          const bulkIndexResult = await searchAfterAndBulkIndex(
-            noReIndexResult,
-            params,
-            services,
-            logger,
-            alertId
-          );
+        const bulkIndexResult = await searchAfterAndBulkIndex({
+          someResult: noReIndexResult,
```

Comment:
I should rename this parameter to be more descriptive.. I'll add that in my open PR.